### ISSUE TITLE
[Error] Save changes in config.yml

### ISSUE THREAD

**Issue by MyThirdAccount** [sstativa/mfc-node#56]
When I _include_ a model, my app fails to save the model to config.yml.

```
[04/08/2018 - 19:38:32] Server listening on: 0.0.0.0:9080
[2018/04/08 - 19:38:33] Connecting to MyFreeCams websocket server xchat106...
[2018/04/08 - 19:38:33] Login handshake completed. Logged in as 'Guest11397' with sessionId 352068565
[04/08/2018 - 19:38:34] [DEBUG] Start new cycle
[04/08/2018 - 19:38:34] 854 model(s) online
[04/08/2018 - 19:38:34] [DEBUG] 0 model(s) in the queue
[04/08/2018 - 19:38:34] [DEBUG] 0 model(s) in config
[04/08/2018 - 19:38:34] [DEBUG] 0 model(s) to capture
[04/08/2018 - 19:38:34] Done, will search for new models in 30 second(s).
[04/08/2018 - 19:38:43] [DEBUG] 13283832 to include
[04/08/2018 - 19:39:04] [DEBUG] Start new cycle
[04/08/2018 - 19:39:04] 851 model(s) online
[04/08/2018 - 19:39:04] [DEBUG] 1 model(s) in the queue
[04/08/2018 - 19:39:04] [DEBUG] 1 model(s) in config
[04/08/2018 - 19:39:04] [DEBUG] 1 model(s) to capture
[04/08/2018 - 19:39:04] bridgettexoxo is now online, starting capturing process
[04/08/2018 - 19:39:04] [DEBUG] Save changes in config.yml
[04/08/2018 - 19:39:04] [ERROR] { Error: EPERM: operation not permitted, open 'E:\mfc-node\config.yml'
  cause:
   { Error: EPERM: operation not permitted, open 'E:\mfc-node\config.yml'
     errno: -4048,
     code: 'EPERM',
     syscall: 'open',
     path: 'E:\\mfc-node\\config.yml' },
  isOperational: true,
  errno: -4048,
  code: 'EPERM',
  syscall: 'open',
  path: 'E:\\mfc-node\\config.yml' }
[04/08/2018 - 19:39:04] Done, will search for new models in 30 second(s).
```

**Comment by sstativa** [comment]
I suppose this is a permissions issue, check that you have permissions to write in this file.

**Comment by MyThirdAccount** [comment]
It's weird. Everything has worked except for saving models to the config file. It will record and it will convert. I modified the security settings for config.yml and gave _Users_ _Full control_ and that didn't fix it for me.

**Comment by MyThirdAccount** [comment]
Ok I think I resolved this issue. I found the following post on Stack Overflow which clued me into something being weird with source control:
[https://stackoverflow.com/questions/33419474/node-fs-error-eperm-operation-not-permitted-open](https://stackoverflow.com/questions/33419474/node-fs-error-eperm-operation-not-permitted-open)

I checked the properties of the file and it was not marked _Read-only_. So what I did was create a brand new config.yml file with the same content and deleted the one that came down from github. This worked.
